### ISSUE TITLE
Try to push xen tools to major linux distribution repositories

### ISSUE THREAD

**Issue by stormi** [xcp-ng/xcp#157]
Few have them, and I don't know if it's because no one packaged them or because it's too hard to select a single version that would work with a large panel of Xen versions.

To be investigated. Help welcome if someone wants to give it a go.

**Comment by Conan-Kudo** [comment]
@stormi I'm pretty sure the reason is because no one did it. It'd be nice to see this get into Fedora (as the upstream for RHEL/CentOS, it should be relatively easy to adapt the packaging for and push it in there).

The process to become a package maintainer and submit packages into the distribution is [well-specified and easy to follow](https://fedoraproject.org/wiki/Join_the_package_collection_maintainers#How_to_join_the_Fedora_Package_Collection_Maintainers.3F).

**Comment by borzel** [comment]
So how did ubuntu it? Maybe theres a receipe to follow?

**Comment by cheese** [comment]
Do you mean https://github.com/xenserver/xe-guest-utilities ? I can help to get it into Fedora and EPEL.

**Comment by olivierlambert** [comment]
Yes, those. It would be ideal to have them bundled in major distro repo :+1:

**Comment by cheese** [comment]
Fedora review request was just submitted: https://bugzilla.redhat.com/show_bug.cgi?id=1707711

**Comment by olivierlambert** [comment]
Nice! :+1:

**Comment by stormi** [comment]
Looks like the package is close to being accepted in Fedora, and then you'll be able to submit it to EPEL I guess. Good news.

If you want to continue working on those tools, we've got https://github.com/xcp-ng/xcp/issues/73 which I still haven't had time to address. It's not straightforward because the build process needs to produce rpms, tarballs, debs, and stuff it all in an ISO image... Which in turns gets included in a RPM :)

**Comment by stormi** [comment]
So, now Fedora Rawhide has the `xe-guest-utilities` package. Did you try to submit it to EPEL yet?

**Comment by cheese** [comment]
> So, now Fedora Rawhide has the `xe-guest-utilities` package. Did you try to submit it to EPEL yet?

I planned to finish EPEL6 packaging and then push to all Fedora stable and EPEL branches. But then I don't have time to finish that by now. I can push to all Fedora stable branches and EPEL7 first.

**Comment by stormi** [comment]
Fedora stable branches and EPEL 7 now have the `xe-guest-utilities` package. Thanks! Do you plan to keep it updated regularly?
